src/sentry/static/sentry/app/utils/discover/fields.tsx

```diff
@@ -446,6 +446,34 @@ export const TRACING_FIELDS = [
   'epm',
 ];
 
+export enum WebVital {
+  FP = 'measurements.fp',
+  FCP = 'measurements.fcp',
+  LCP = 'measurements.lcp',
+  FID = 'measurements.fid',
+}
+
+const MEASUREMENTS: Readonly<Record<WebVital, ColumnType>> = {
+  [WebVital.FP]: 'duration',
+  [WebVital.FCP]: 'duration',
+  [WebVital.LCP]: 'duration',
+  [WebVital.FID]: 'duration',
+};
+
+const MEASUREMENT_PATTERN = /^measurements\.([a-zA-Z0-9]+)$/;
```

Comment:
underscores, dots, and hyphens are valid

Reply:
fixed in c45a95f